docs/source/index.rst

```diff
@@ -0,0 +1,17 @@
+Welcome to py-libp2p's documentation!
```

Comment:
We have a few general [documentation guidelines](https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/documentation.md), and we're trying to make sure that all our repos are as close to them as possible.

That's why I'm taking this opportunity to point this out so that we can get these policies involved early on.

It's far from perfect but the project closest to the ideals of this guide line is probably Trinity so I recommend to take a [look at its docs.](https://trinity-client.readthedocs.io/en/latest/)

In particular, it would be nice if we could roughly follow this navigation structure.

![image](https://user-images.githubusercontent.com/521109/67662215-3d7a1400-f963-11e9-8de5-feaaea817200.png)

### General
**Introduction**: The landing page, giving a quick overview about the project, additional pointers.
**Quickstart**: Pretty much what you already have with all the installation instructions
**Release Notes**: Points to the towncrier generated release notes

### Fundamentals

**API**: Source generated API docs as you already have them included in the PR
**Cookbook**: Collection of small recipes, examples. Doesn't need to exist in the beginning
**Guides**: Tutorial style guides. It's ok to just have a link to the *Quickstart* here to begin with

### Community

**Contributing**: Instructions on how to contribute to the project
**Code of Conduct**: The code of conduct that we use across repos